tests/e2e/utils.js

```diff
@@ -448,14 +448,10 @@ export const openBlockEditorSettings = async () => {
 	}
 };
 
-/**
- * Click a link and wait for the page to load.
- *
- * @param {string} selector The CSS selector of the link to click.
- */
-export const clickLink = async ( selector ) => {
-	await Promise.all( [
-		page.click( selector ),
-		page.waitForNavigation( { waitUntil: 'networkidle0' } ),
-	] );
+export const waitForAllProductsBlockLoaded = async () => {
```

Comment:
Small nitpick, not blocking:

1. Could you add a little JSDoc to this? I know the name is self explanatory, but it might be worth explaining what it does and it also encourages contributors to always put JSDocs on their functions.
2. I know a lot of selectors here are hard-coded, but I would like to see it extracted somewhere else, if you have the patience for that 😊